steps/data_scraping_steps/scrape_mind_data/scrape_mind_data_step.py

```diff
@@ -30,12 +30,33 @@
     ],
 }
 
+URLS_TO_DISCARD = [
+    "https://www.mind.org.uk/information-support/types-of-mental-health-problems/depression/",
+    "https://www.mind.org.uk/information-support/drugs-and-treatments/antidepressants-a-z/overview/",
+    "https://www.mind.org.uk/information-support/types-of-mental-health-problems/recreational-drugs-alcohol-and-addiction/",
+    "https://www.mind.org.uk/information-support/drugs-and-treatments/complementary-and-alternative-therapies/",
+    "https://www.mind.org.uk/information-support/drugs-and-treatments/sleeping-pills-and-minor-tranquillisers-a-z/overview/",
+    "https://www.mind.org.uk/information-support/drugs-and-treatments/antipsychotics-a-z/overview/",
+]
+
 
 class Scraper:
     """This class aims to scrape data from the "Types of mental health problems", "Drugs and treatments" and the sub-sections from the "Helping someone else" as they all share the same HTML structure."""
 
     session = HTMLSession()
 
+    def __init__(self, urls_to_discard=None) -> None:
+        """Initialise Mind data scraper.
+
+        Args:
+            urls_to_discard (List[str]): list of URLs to discard from the results
```

Comment:
Nit: a clarifying point could be added here to say that it overwrites pre-configured URLs for mind.